### PR TITLE
rAmber deployments

### PR DESCRIPTION
**Pull request by 0xZenus**

### CODE REVIEW

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 1]
```diff
@@ -760,6 +769,8 @@ amberUsdcReyaLmMaxStaleDuration = "0"
 pool1_quoteTokenTargetRatioUnscaled = "0.2"
 pool1_DEUSDTargetRatioPostQuoteUnscaled = "0.4"
 pool1_SDEUSDTargetRatioPostQuoteUnscaled = "0.6"
+pool1_RSELINITargetRatioPostQuoteUnscaled = "0"
```
Comment (arturbeg) [inline packages/tomls/src/omnibus/reya_cronos.toml:772]: shall we go straight for the target ratios proposed in team chat?

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 2]
```diff
@@ -1645,3 +1656,15 @@ cp1Rusd_rseliniParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
 cp1Rusd_rseliniParentConfig_priceHaircutUnscaled = "0"
 cp1Rusd_rseliniParentConfig_autoExchangeDiscountUnscaled = "0"
 cp1Rusd_rseliniParentConfig_oracleNodeId = "<%= settings.seliniUsdcReyaLmNodeId %>"
+# -----------------------------------------------
+# TODO later: pass depositingEnabled through var (but there's no parseBool)
+# cp1Rusd_rusd_depositingEnabled = "true"
+cp1Rusd_ramber_capUnscaled = "<%= formatEther(MaxUint256) %>"
```
Comment (arturbeg) [inline packages/tomls/src/omnibus/reya_cronos.toml:1662]: hmm, shall we go ahead and set it to 2m (same in mainnet)?

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 3]
```diff
@@ -1645,3 +1656,15 @@ cp1Rusd_rseliniParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
...
+cp1Rusd_ramber_autoExchangeThresholdUnscaled = "0"
```
Comment (arturbeg) [inline packages/tomls/src/omnibus/reya_cronos.toml:1663]: hmm, actually let's flag in the team chat that we will need to make an explicit decision on the haircuts / discounts that we want to use for the ramber and rselini token, there're a few neuance s within this that will be good to discuss

one thing worth noting is that the effectively liquidity / max exposure the pool can support in perps trading will decrease as a result of introducing ramber and rselini if we have haircuts of zero

File: packages/tomls/src/omnibus/reya_network.toml [hunk 4]
```diff
@@ -748,6 +757,9 @@ amberUsdcReyaLmMaxStaleDuration = "0"
 pool1_quoteTokenTargetRatioUnscaled = "0.2"
 pool1_DEUSDTargetRatioPostQuoteUnscaled = "0.4"
 pool1_SDEUSDTargetRatioPostQuoteUnscaled = "0.6"
+# TODO: change
```
Comment (arturbeg) [inline packages/tomls/src/omnibus/reya_network.toml:760]: shall we go ahead and change it in this batch of configs?

**Review by arturbeg — DISMISSED** [review]
looks good, left smol comments

**Review by avniculae — APPROVED** [review]

**Pull request merged**